src/Aide.ClinicalReview.Service/Handlers/AideClinicalReviewRequestCallBackHandler.cs

```
@@ -94,7 +94,15 @@ public async Task HandleMessage(JsonMessage<AideClinicalReviewRequestMessage> me
 
             await _clinicalReviewRepository.CreateAsync(clinicalReviewRecord);
 
-            await this.HandleTaskDetails(message.Body.ExecutionId, message.Body.Files, message.Body.ReviewerRoles.ToList());
+            try
+            {
+                await this.HandleTaskDetails(message.Body.ExecutionId, message.Body.Files, message.Body.ReviewerRoles.ToList());
+            }
+            catch (Exception ex)
+            {
+                _logger.LogError(exception: ex, "Handling task details errored");
+                throw;
```

Do we want to re-throw the error? Just wondering what the response will look like then

---

this should propagate the exception. this way we are not changing the call stack of the exception. if we did `throw ex;` the call stack will change.